docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -638,7 +638,273 @@ Take the following code for example.
            model = models.__dict__[args.arch]()
 
 ``args.arch`` is a hyperparameter. You should define the hyperparameter space in the
-:doc:`experiment configuration </reference/reference-training/experiment-config-reference>` and use
-``self.context.get_hparams()``, which gives you access to all the hyperparameters for the current
-trial. By doing so, you get better tracking in the WebUI, especially for experiments that use a
-searcher.
+:doc:`experiment configuration </reference/reference-training/experiment-config-reference>`. By
+doing so, you get better tracking in the WebUI, especially for experiments that use a searcher.
+Depending on how your trial is run, you can access all the current hyperparameters from inside the
+trial by either calling ``self.context.get_hparams()`` if you submitted your trial with
+``entrypoint: model_def:Trial`` or passing in hyperparameters directly into the Trial ``__init__`` if
+using PyTorch Trainer API.
+
+************************
+ Trainer API
+************************
+
```

Comment:
i reorganized this page to make it look like two sections of the same doc (PyTorch Trial, PyTorch Trainer)

commit SHA f1a1e8513949d383de5622c9538c37355424ff63